format/tzif/testdata/generate_fqtest.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
👍 i appreciate the fqtest thing more and more, makes it so easy to do things like this